lookup/functions/lookup.js

```diff
@@ -26,12 +26,21 @@ exports.handler = async function (context, event, callback) {
 
     const types = typeof event.types === 'object' ? event.types : [event.types];
     const client = context.getTwilioClient();
-    const pn = await client.lookups
+
+    const resp = await client.lookups
       .phoneNumbers(event.phone)
       .fetch({ type: types });
 
+    if (types.includes('lti')) {
+      const { lineTypeIntelligence } = await client.lookups.v2
```

Comment:
I feel like I'm missing something here. Why does this do an extra API request for line type intelligence rather than just use the v2 API for all responses?

Reply:
Argh, I hadn't read the actual description. Disregard me!